instrumentation/log4j/log4j-context-data/log4j-context-data-2.17/library-autoconfigure/src/main/java/io/opentelemetry/instrumentation/log4j/contextdata/v2_17/OpenTelemetryContextDataProvider.java

```diff
@@ -21,6 +25,9 @@
  * #supplyContextData()} is called when a log entry is created.
  */
 public class OpenTelemetryContextDataProvider implements ContextDataProvider {
+  private static final boolean BAGGAGE_ENABLED =
+      ConfigPropertiesUtil.getBoolean(
+          "otel.instrumentation.log4j-context-data.baggage-enabled", false);
```

Comment:
The same property in the Logback MDC instrumentation is called `...add-baggage`; can we keep the names consistent?
```suggestion
          "otel.instrumentation.log4j-context-data.add-baggage", false);
```